logistics_planning_base/models/stock_move.py

```diff
@@ -0,0 +1,10 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    logistics_schedule_id = fields.Many2one('logistics.schedule')
```

Comment:
Could you add a new field named `logistics_schedule_disabled`?
- This will be used for stock move domain selection in logistics schedule records, adding it to existing condition.
- By default as `False` (then, record is available).
- Only editable when stock move record is in done state and no `logistics_schedule_id` is linked with it yet. The goal is determine if a stock move is available (or not) for logistics module only when move is finished and is still unused.
- Shown in views for its management.
- Within installation, a hook method (it could be a post init SQL statement) will set every done stock move as `True`, so when this addon is installed, no stock moves could be selected yet (history moves set could be huge and we can avoid it).